content/guides/migrating-to-cypress/protractor.md

```diff
@@ -1170,7 +1170,7 @@ Read the docs on
 [recording test results](http://on.cypress.io/recording-project-runs) to the
 [Cypress Dashboard](/guides/dashboard/introduction) to learn more.
 
-### Specifying a custom `cypress.json` config file
+### Specifying a custom configuration file
```

Comment:
```suggestion
### Specifying a custom Cypress configuration file
```

Reply:
You also need to change the `configFile` option in the example following this to something like `cypress.production.config.js`